Sources/Errors/GenericErrorViewModel.swift

```diff
@@ -0,0 +1,33 @@
+import GDSAnalytics
+import GDSCommon
+import Logging
+
+struct GenericErrorViewModel: GDSErrorViewModel, BaseViewModel {
+    let image: String = "exclamationmark.circle"
+    let title: GDSLocalisedString = "Something went wrong"
```

Comment:
Should these strings be localised?

Reply:
Ideally yes, but haven't got our ticket to implement localisation yet 🙁